libirods_rule_engine_plugin-indexing.cpp

```diff
@@ -697,6 +740,151 @@ namespace {
         return retvalue.size() ? retvalue : "application/octet-stream";
     }
 
+    class PermsCalc {
+        std::map<std::string,std::string> users,groups;
+        std::map<std::string,std::list<std::string>> members;
+        std::map<std::string,int> user_entry;
+        std::multimap<int,std::string> user_perms, group_perms;
+
+        static std::map<int,std::string> perm_names;
```

Comment:
add space after commas

use a trailing underscore for private member variables